projects/1.20/assets/productivetrees/productivetrees/lang/zh_cn-base.json

```
@@ -0,0 +1,497 @@
+    "item.productivetrees.clove": "丁香蒲桃",
+    "item.productivetrees.coconut": "椰子",
+    "item.productivetrees.coffee_bean": "咖啡豆",
+    "item.productivetrees.copoazu": "大花可可",
+    "item.productivetrees.cork": "软木",
+    "item.productivetrees.cured_rubber": "硫化橡胶",
+    "item.productivetrees.date": "椰枣",
+    "item.productivetrees.date_palm_juice": "椰枣汁",
+    "item.productivetrees.dracaena_sap": "龙血树脂",
+    "item.productivetrees.elderberry": "接骨木莓",
+    "item.productivetrees.fig": "无花果",
+    "item.productivetrees.finger_lime": "指橙",
+    "item.productivetrees.flowering_crabapple": "垂丝海棠",
+    "item.productivetrees.fustic": "黄颜染料",
+    "item.productivetrees.ginkgo_nut": "银杏果",
+    "item.productivetrees.golden_delicious_apple": "黄苹果",
+    "item.productivetrees.granny_smith_apple": "青苹果",
+    "item.productivetrees.haematoxylin": "苏木",
+    "item.productivetrees.hala_fruit": "露兜果",
+    "item.productivetrees.haw": "山楂",
+    "item.productivetrees.hazelnut": "榛子",
+    "item.productivetrees.juniper_berry": "杜松实",
+    "item.productivetrees.key_lime": "来檬",
+    "item.productivetrees.kumquat": "金橘",
+    "item.productivetrees.lemon": "柠檬",
+    "item.productivetrees.lime": "酸橙",
+    "item.productivetrees.mandarin": "柑橘",
+    "item.productivetrees.mango": "芒果",
+    "item.productivetrees.maple_sap_bucket": "枫树汁液",
+    "item.productivetrees.maple_syrup": "枫糖",
+    "item.productivetrees.nectarine": "油桃",
+    "item.productivetrees.nutmeg": "肉豆蔻",
+    "item.productivetrees.olive": "橄榄",
+    "item.productivetrees.orange": "橙子",
+    "item.productivetrees.osange_orange": "桑橙",
+    "item.productivetrees.papaya": "木瓜",
+    "item.productivetrees.peach": "桃子",
+    "item.productivetrees.pear": "梨子",
+    "item.productivetrees.pecan": "山胡桃",
+    "item.productivetrees.persimmon": "柿子",
+    "item.productivetrees.pistachio": "开心果",
+    "item.productivetrees.planet_peach": "星辰桃",
+    "item.productivetrees.plantain": "芭蕉",
+    "item.productivetrees.plum": "李子",
+    "item.productivetrees.pollen": "花粉",
+    "item.productivetrees.pomegranate": "石榴",
+    "item.productivetrees.pomelo": "文旦柚",
+    "item.productivetrees.prairie_crabapple": "草原海棠",
+    "item.productivetrees.red_banana": "红香蕉",
+    "item.productivetrees.roasted_almond": "烤扁桃",
+    "item.productivetrees.roasted_beechnut": "烤山毛榉坚果",
+    "item.productivetrees.roasted_brazil_nut": "烤巴西栗",
+    "item.productivetrees.roasted_butternut": "烤灰胡桃",
+    "item.productivetrees.roasted_candlenut": "烤石栗",
+    "item.productivetrees.roasted_cashew": "烤腰果",
+    "item.productivetrees.roasted_chestnut": "烤板栗",
+    "item.productivetrees.roasted_coffee_bean": "烤咖啡豆",
+    "item.productivetrees.roasted_ginkgo_nut": "烤银杏果",
+    "item.productivetrees.roasted_hazelnut": "烤榛子",
+    "item.productivetrees.roasted_pecan": "烤山胡桃",
+    "item.productivetrees.roasted_pistachio": "烤开心果",
+    "item.productivetrees.roasted_walnut": "烤胡桃",
+    "item.productivetrees.rowan": "八角",
```

```suggestion
    "item.productivetrees.rowan": "花楸果",
```
这个是啥，需要和上方的“花楸果”保持一致吗